### PR TITLE
Milestone 3: Styling

### PR DESCRIPTION
**Pull request by nicolasmarino99**
In this project we add:
- styles to our components
- pass props to style components

### CODE REVIEW

File: src/styles/App.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  height: 100%;
+}
+
+#root {
+  height: 100%;
+  background: rgb(2, 0, 36);
+  background: -moz-linear-gradient(27deg, rgba(2, 0, 36, 1) 0%, rgba(9, 9, 121, 1) 35%, rgba(0, 212, 255, 1) 100%);
+  background: -webkit-linear-gradient(27deg, rgba(2, 0, 36, 1) 0%, rgba(9, 9, 121, 1) 35%, rgba(0, 212, 255, 1) 100%);
+  background: linear-gradient(27deg, rgba(2, 0, 36, 1) 0%, rgba(9, 9, 121, 1) 35%, rgba(0, 212, 255, 1) 100%);
+  //filter: progid:dximagetransform.microsoft.gradient(startColorstr="#020024", endColorstr="#00d4ff", GradientType=1);
```
Comment (Forison) [inline src/styles/App.scss]: Kindly remove commented code

File: src/styles/ButtonPanel.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.ButtonPanel {
+  height: 100%;
+
+  .row-container {
+    height: 20%;
+    //width: 100%;
```
Comment (Forison) [inline src/styles/ButtonPanel.scss]: Kindly remove commented code

File: src/styles/ButtonPanel.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.ButtonPanel {
+  height: 100%;
+
+  .row-container {
```
Comment (Forison) [inline src/styles/ButtonPanel.scss:2]: height is 100px ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/rowHeight.png))

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Changes Required❌ 
# 🙋‍♂️ Nico.
#### You have done a great job so far, however, there are a few technical issues we have to address to get this project approved😎.
* [x] Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
* [x]  It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- Kindly check your stylesheets.
* [x] each row's height is 100px ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/rowHeight.png))
* [x] it accepts `color` and `wide` as an incoming props ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorAndWideProps.png))
* [x] Default `color` for button is orange ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png))
* [x] `wide` is boolean, true means that button take 50% of space (used for `0` button) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png))
- You can maintain size as you have and change it to a boolean.
### That is it kindly make the changes and submit the project for a re-review🙍‍♂️.
#### Happy coding👨‍💻
Addo Forison

**Comment by nicolasmarino99** [comment]
For some reason, I have an Eslint error. I have checked and everything seems to work fine on my local computer.

**Review by bellom — APPROVED** [review]
## Hi @nicolasmarino99  :clap: 

**Great work making all the necessary corrections from the previous review.**

**Note:**
1. To solve the linter's error, kindly look into this;
https://stackoverflow.com/questions/40271230/how-to-run-eslint-fix-from-npm-script
2. **Kindly update your `README` file before merging to `dev`**
Checkout out the below link for help, free free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/


## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**